acc_test/test_utils.go

```diff
@@ -54,3 +56,26 @@ func checkClusterRunning(resourceName, hostname, username, apiKey string) resour
 		return nil
 	}
 }
+
+func addDCtoCluster(resourceName, hostname, username, apiKey, requestBody string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		resourceState := s.Modules[0].Resources["instaclustr_cluster."+resourceName]
+		id := resourceState.Primary.Attributes["cluster_id"]
+		client := new(instaclustr.APIClient)
+		client.InitClient(hostname, username, apiKey)
+		body, _ := ioutil.ReadFile(requestBody)
+
+		// Adding a new DC to an existing cluster
+		url := fmt.Sprintf("%s/provisioning/v1/%s/cluster-data-centres", hostname, id)
+		resp, err := client.MakeRequest(url, "POST", body)
+
+		if err != nil {
+			return err
+		}
+		bodyText, err := ioutil.ReadAll(resp.Body)
```

Comment:
Should we return an error in the case of a failure to read the response body here?

Reply:
Yep added return err below